lingo.yml

```diff
@@ -30,6 +30,6 @@ checkers:
     tab_width: 4
   left_quantifiers:
   func_cyclo:
-    max: 15
+    max: 20
```

Comment:
Why do we need to increase the cyclomatic complexity?

Reply:
https://github.com/s2gatev/lingo/pull/24/files#diff-be28c70f8d6da8565c996817f981df07R68

Reply:
Since you extracted the `funcTypeName` helper, I believe the file is still within the 15.

Reply:
Yes, updated.